jobs/build/advisories/advisories.groovy

```diff
@@ -0,0 +1,70 @@
+buildlib = load("pipeline-scripts/buildlib.groovy")
+commonlib = buildlib.commonlib
+
+ADVISORIES = [:]
+
+def create_advisory(name) {
+    impetus = "${params.IMPETUS}"
+    if (name ==~ /.*rhsa$/) {
+        impetus = "cve"
+    } else if (name == "metadata") {
+        impetus = "metadata"
+    } else if (name == "extras") {
+        impetus = "extras"
+    }
+
+    kind = "image"
+    if (name ==~ /rpm/) {
+        kind = "rpm"
+    }
+
+    type = "RHBA"
+    if (name ==~ /rhsa/) {
+        type = "RHSA"
```

Comment:
creating an RHSA currently requires specifying for what, and adds the flaw bug in addition to the tracker (which doesn't happen with an ordinary add), so there's some complexity around it.

i actually think... we are going to end up turning our regular advisories into RHSAs as needed rather than separating out the CVEs. so this expansion may not be necessary.

Reply:
Is the requirement of specifying what for by elliott, or by Errata Tool?

Reply:
current this will not work because specifying bug is needed, @vfreex do you think we need to add an option like `--mode cve` to `find-bugs` then remove this limitation? 
```
raise ElliottFatalError(
"When creating an RHSA, you must provide a list of bug id(s) using one or more `--bug` options.")
```

Reply:
@shiywang I am ok with removing the limitation, but since @sosiouxme proposed we can attach all bugs (no matter it is CVE or not) to an RHBA then convert it to RHSA later, I would like to hold off this change until we decide to do that.

Reply:
right, it's an elliott requirement not an ET requirement. but in any case i think we will be re-working elliott a bit regarding CSVs and not need the ready-made RHSAs.